index.html

```diff
@@ -1910,6 +1910,80 @@
       </pre>
       <section>
         <h2>
+          Creating a <code>PaymentAddress</code>
+        </h2>
+        <p>
+          The steps to <dfn>create a payment address</dfn> are given by the
+          following algorithm.
```

Comment:
It'd be nice to mention in this sentence something like "**create a payment address** from user-provided input" as otherwise you get a bit taken aback realizing that this algorithm isn't a general creation algorithm, it's specifically about user input.